package/setup.py

```diff
@@ -286,27 +314,27 @@ def detect_openmp():
                     'MDAnalysis.visualization'],
           package_dir={'MDAnalysis': 'MDAnalysis'},
           ext_package='MDAnalysis',
-          ext_modules=extensions,
+          ext_modules=extensions(config),
           classifiers=CLASSIFIERS,
           long_description=LONG_DESCRIPTION,
           cmdclass=cmdclass,
           # all standard requirements are available through PyPi and
           # typically can be installed without difficulties through setuptools
           install_requires=[
-              'numpy>=1.5.0',  # currently not useful because without numpy we don't get here
-              'biopython>=1.59',  # required for standard PDB reader and sequence alignment
-              'networkx>=1.0',  # LeafletFinder
-              'GridDataFormats>=0.2.2',  # volumes and densities
+              'biopython>=1.59',
+              'networkx>=1.0',
+              'GridDataFormats>=0.2.2',
           ],
           # extras can be difficult to install through setuptools and/or
           # you might prefer to use the version available through your
           # packaging system
           extras_require={
-              'AMBER': ['netCDF4>=1.0'],  # for AMBER netcdf, also needs HDF5 and netcdf-4
+              'AMBER': ['netCDF4>=1.0'],  # for AMBER netcdf, also needs HDF5
+                                          # and netcdf-4
               'analysis': [
                   'matplotlib',
-                  'scipy',  # sparse contact matrix
```

Comment:
We had debated this in the past and had decided that it is more important to be able to install the core with `pip install MDAnalysis` than to be guarantee that *all* the analysis modules run. The problem with scipy and matplotlib is that they don't install nicely (at least I was never able to automatically pip-install them). The same applies to netcdf (which is needed inside the "core" for AMBER). It's a compromise; I haven't heard complaints about it yet whereas I fielded lots of questions when we still had the difficult dependencies. Not everyone has the full scientific python stack installed, especially when you're fairly new to python and just heard from a colleague that "it is so easy to just do this in MDAnalysis"...